setup.py

```diff
@@ -20,6 +20,9 @@
 ARCTICDB_USING_CONDA  = os.environ.get("ARCTICDB_USING_CONDA", "0")
 ARCTICDB_USING_CONDA = ARCTICDB_USING_CONDA != "0"
 
+ARCTICDB_CPU_COUNT  = os.environ.get("ARCTICDB_CPU_COUNT", "0")
+ARCTICDB_CPU_COUNT = int(ARCTICDB_CPU_COUNT)
```

Comment:
Can we rename this `ARCTICDB_BUILD_CPU_COUNT` - there's already an environment variable (that looks very different) that controls how many threads/cores to use at runtime and I don't want to confuse the two at a glance.

Reply:
I am also in favor of using `CMAKE_BUILD_PARALLEL_LEVEL` here.